docs/websocket/feedback.md

```diff
@@ -0,0 +1,131 @@
+
+
+## Prepare
+
+Prepare a new speech feedback. This RPC should be called for each new speech
+feedback. A unique id is generated for a speech feedback and a speech challenge
+is prepared.
+
+### URI
+
+```
+nl.itslanguage.feedback.prepare
+```
+
+### Parameters
+
+Name            | Type     | Description
+----------------|----------|------------
+challenge_id    | `string` | **Required** The id of the speech challenge to prepare.
+
+### Response
+
+The id of the new speech recording is returned as a string.
+
+
+## Start listening
+
+In order to receive feedback the server needs to listen for audio on the
+registered audio rpc. While listening the server will reply using progressive
+results.
+The server will stop listening when the audio rpc returns.
+
+### URI
+
+```
+nl.itslanguage.feedback.listen_and_reply
+```
+
+### Parameters
+
+Name         | Type     | Description
+-------------|----------|------------
+recording_id | `string` | **Required** The unique id of the speech recording.
+rpc          | `string` | **Required** The URI of a [registered audio rpc](wamp.md#register-audio-procedure).
+
+### Response
+
+The rpc returns progressive results for realtime feadback. After every sentence
+the following json is sent as a progressive result:
+
+```json
+{
+  "sentence": 0,
+  "feedback": true
+}
+```
+
+Name     | Type   | Description
+---------|--------|------------
+sentence | `int`  | The index of the sentence, starting at `0`.
+feedback | `bool` | Feedback score, `false` means bad and `true` means good.
+
+When the recording is finished a recording with feedback is returned:
+
+```json
+{
+  "id": "recording_1",
+  "created": "2014-01-28T21:25:10Z",
+  "updated": "2014-01-28T21:25:10Z",
+  "audioUrl": "https://api.itslanguage.nl/download/audio.wav",
+  "feedback": [
+    false,
+    true,
+    true
+  ]
+}
+```
+
+Name     | Type     | Description
+---------|----------|------------
+id       | `string` | The id of the recording.
+created  | `string` | The timestamp when the recording was created.
+updated  | `string` | The timestamp when the recording was last updated.
+audioUrl | `string` | The url to fetch the recorded audio.
+feedback | `array`  | A list containing feedback (`bool`) per sentence.
+
+
+## Pause
+
+When desired the feedback recording can be paused. Doing so will stop the
+server from processing the audio stream and returning feedback.
+
+### URI
+
+```
+nl.itslanguage.feedback.pause
+```
+
+### Parameters
+
+Name | Type     | Description
+-----|----------|------------
+id   | `string` | **Required** The id of the feedback to pause.
+
+
+## Resume
+
+A paused feedback can be resumed using this RPC. After this the audio stream is
+again processed by the server.
+
+### URI
+
+```
+nl.itslanguage.feedback.resume
+```
+
+### Parameters
+
+Name        | Type     | Description
+------------|----------|------------
+id          | `string` | **Required** The id of the feedback to pause.
```

Comment:
I think you mean `resume` instead of `pause`

Reply:
Good catch!